src/test/Components/ImagesTable/ImagesTable.test.js

```diff
@@ -51,6 +51,37 @@ const store = {
                     }
                 },
             },
+            '77fa8b03-7efb-4120-9a20-da66d68c4494': {
```

Comment:
Bit confused with the diff in this file, did you just move it up ?

Reply:
ah ya I moved it when playing around with the tests. I can move it back to improve the diff since the location doesn't matter.